mars/core.py

```diff
@@ -432,6 +443,32 @@ def build_graph(self, graph=None, cls=DAG, tiled=False, compose=True, executed_k
                           if c not in visited])
         if tiled and compose:
             graph.compose(keys=keys)
+
+        def adapt_tensor(tensor):
```

Comment:
I think the name of this nested function can be more explicit, for instance, ``remove_tiled``. ``adapt`` is too general.